ledger/bulletin.go

```diff
@@ -105,7 +110,8 @@ func (b *bulletin) notifyRound(rnd basics.Round) {
 		}
 
 		delete(b.pendingNotificationRequests, pending)
-		signal.notify()
+		// signal the channel by closing it; this is under lock and will only happen once
+		close(signal.signal)
```

Comment:
I feel like if you need the comment, I'd support keeping this as a `signal.notify()` function that'll just get inlined anyway.